Assets/Scripts/MovementInput.cs

```diff
@@ -61,10 +63,15 @@ public class MovementInput : MonoBehaviour {
 	}
 
   private void Rotate() {
-    transform.rotation = Quaternion.Slerp(transform.rotation, Quaternion.LookRotation(moveDirection), rotationSpeed);
+    transform.rotation = Quaternion.Lerp(
+      transform.rotation, 
+      new Quaternion(0, playerCamera.transform.rotation.y, 0, playerCamera.transform.rotation.w),
+      rotationSpeed
+    );
   }
 
   private void Animate() {
-    animator.SetFloat("Speed", moveDirection.sqrMagnitude);
+    animator.SetFloat("SpeedX", inputX);
+    animator.SetFloat("SpeedY", inputZ);

```

Comment:
These parameters are not being used by the Animator. Maybe missed saving the scene or the prefab?